common/src/Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet/tests/Default/SchemaValidationTests.cs

```diff
@@ -8,24 +8,64 @@ namespace Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet.Tests.Default {
     using System.IO;
     using System.Linq;
     using System.Text;
+    using System.Text.Json;
     using Xunit;
 
     /// <summary>
-    /// These tests are designed to exercise the 
-    /// generated schema for configuration file 
-    /// validation. These tests are additionally 
-    /// found in the schema generator tool located
+    /// These tests are designed to exercise the generated schema for configuration file 
+    /// validation. These tests are additionally found in the schema generator tool located
     /// at: https://github.com/WilliamBerryiii/opcpublisherschemavalidator
+    /// 
+    /// The referenced schema file is a linked asset in the project file set to 
+    /// copy to the output build directory so that it can be easily referenced here.
     /// </summary>
     public class SchemaValidationTests {
 
+        // allow training commas in the test parsers as the main serializer for 
+        // this solution supports training commas. 
+        private static JsonDocumentOptions parseOptions = new JsonDocumentOptions {
+            AllowTrailingCommas = true,
+        };
+
         [Fact]
         public void EnsureTestConfigurationPassesSchemaValidation() {
-            using var schemaReader = new StreamReader("Storage/publishednodesschema.json");
+            using var schemaReader = new StreamReader("Default/publishednodesschema.json");
             var validator = new JsonSchemaDotNetSchemaValidator();
-            var results = validator.Validate(Encoding.UTF8.GetBytes(pn), new StringReader(schema));
+            var results = validator.Validate(Encoding.UTF8.GetBytes(testConfiguration), schemaReader);
             Assert.True(results.All(r => r.IsValid));
+        }
+
+        [Fact]
+        public void ByteStringNodeIdsThatAreNotByteStingsReturnErrors() {
+            using var schemaReader = new StreamReader("Default/publishednodesschema.json");
+            // break the id of the first Node to ensure an error is thrown
```

Comment:
nit: probably want to capitalise comments and add blank line above them.

Reply:
resolved in next commit.

Reply:
Will do.

Reply:
Commit https://github.com/WilliamBerryiii/Industrial-IoT/pull/2/commits/887860fa4a8fbe0cdc54ae09f79286d4f38d296c has the resolution for this.